tensorflow/lite/micro/micro_mutable_op_resolver.h

```diff
@@ -30,6 +30,8 @@ limitations under the License.
 #include "tensorflow/lite/schema/schema_generated.h"
 
 namespace tflite {
+TfLiteRegistration* Register_ETHOSU();
+const char* GetString_ETHOSU();
```

Comment:
move these to a new header `micro/kernels/ethous.h`, similar to `micro/kernels/fully_connected.h`

Reply:
Done